apps/web/src/components/Common/CategoryFilters.tsx

```diff
@@ -38,14 +38,31 @@ const CategoryFilters = () => {
 
   const slide = (shift: number) => {
     Analytics.track(TRACK.CLICK_CATEGORIES_SCROLL_BUTTON)
+
     if (scrollRef.current) {
+      /**
```

Comment:
Great work!

Thanks for the comments to convey. We can clean up the comments 👍🏻

Reply:
Thanks for the feedback, I will remove the comments and set the variable type to `const` in the next PR.

Should I change the filename in this one itself or make a new PR for it?

Reply:
Update all comments in this same PR!